.swiftlint.yml

```diff
@@ -0,0 +1,16 @@
+disabled_rules:
+  - line_length
+  - force_try
+  - force_cast
+
+included:
+  - Sources
+  - Tests
+
+type_body_length:
+  - 700 #warning
```

Comment:
I referred to https://github.com/krzysztofzablocki/Sourcery/blob/master/.swiftlint.yml#L26.
If there is no rule about `type_body_length ` and `file_length`, we have an error.

```console
error: Type Body Length Violation: Type body should span 350 lines or less excluding comments and whitespace: currently spans 388 lines (type_body_length)
warning: File Line Length Violation: File should contain 400 lines or less: currently contains 784 (file_length)
```

How about the following rule?
```yaml
type_body_length:
  - 400 #warning

file_length:
  - 800 #warning
```
Or we can disable them by using `swiftlint:disable ***`